src/scenic/simulators/Gazebo_sawyer/demos/example.scenic

```diff
@@ -0,0 +1,49 @@
+model scenic.simulators.Gazebo_sawyer.model
```

Comment:
This entire file should be moved to `examples/gazebo_sawyer/example.scenic`. Also, we should create an `example/gazebo` directory and place some example files under there as well!